request-manager/graph/grapher.go

```diff
@@ -287,30 +310,29 @@ func buildSeqGraph(seqSpec *spec.Sequence, idgen id.Generator) (seqGraph *Graph,
 		}
 	}
 
-	// Append sink node to ordering
 	seqGraph.Order = append(seqGraph.Order, seqGraph.Sink)
 
 	// Make sure we haven't created a deformed graph
 	// If we do, it's a bug in the code, not a problem with the specs
 	if !seqGraph.IsValidGraph() {
-		return nil, nil, fmt.Errorf("malformed graph created")
+		return nil, nil, fmt.Errorf("malformed sequence graph created")
 	}
 
 	return seqGraph, jobArgs, nil
 }
 
-func newGraph(name string, idgen id.Generator) (*Graph, error) {
+func newSeqGraph(name string, idgen id.Generator) (*Graph, error) {
 	id, err := idgen.UID()
 	if err != nil {
 		return nil, err
 	}
-	source := newNoopNode(name+"_start", id)
+	source := newNoopSeqNode(name+"_start", id)
```

Comment:
Would it break anything if we did s/start/begin?  Imho the standard pairs of terms are begin/end and start/stop (and source/sink in this context).